src/contact-form/index.tsx

```diff
@@ -0,0 +1,130 @@
+import style from './contact-form.module.css';
+// import React, { SyntheticEvent, useState } from 'react';
+import React from 'react';
+import { EmployeeTile } from '../employees';
+import { EmployeeItem } from '../employees/types';
+
+interface contactFormProps {
+  employees?: EmployeeItem[];
+  lessMargin?: boolean;
+}
+
+export const ContactForm = ({
+  employees, // lessMargin = false,
+}: contactFormProps) => {
+  // const [formData, setFormData] = useState({
```

Comment:
Let's get rid of the code that is commented to keep things clean.

Reply:
I agree, I've removed all commented out code :)